test/nools/task-emitter.spec.js

```diff
@@ -95,6 +95,206 @@ describe('task-emitter', () => {
         });
       });
 
+      it('task priority is not required', () => {
+        // given
+        const contact = personWithReports(aReport());
+        const task = aPersonBasedTask();
+        const config = {
+          c: contact,
+          targets: [],
+          tasks: [ task ],
+        };
+        delete config.tasks[0].priority;
+        
+        // when
+        const { emitted } = runNoolsLib(config);
+
+        // then
+        expect(emitted).to.have.length(2);
+        expect(emitted[0]).to.be.shallowDeepEqual({
+          '_type': 'task',
+          'actions': [{
+            'content': {
+              'contact': {
+                '_id': contact.contact._id,
+                'reported_date': TEST_DATE,
+                'type': 'person'
+              },
+              'source': 'task',
+              'source_id': contact.contact._id
+            },
+            'form': 'example-form',
+            'label': 'Follow up',
+            'type': 'report'
+          }],
+          'contact': {
+            '_id': contact.contact._id,
+            'reported_date': TEST_DATE,
+            'type': 'person'
+          },
+          'date': TEST_DAY,
+          'resolved': false
+        });
+      });
+
+      it('task priority is string set from object', () => {
+        // given
+        const report = aReport();
+        const contact = personWithReports(report);
+        const config = {
+          c: contact,
+          targets: [],
+          tasks: [ aPersonBasedTask() ]
+        };
+        config.tasks[0].priority = {
+          level: 'high',
+          label: [{ locale: 'en', label: 'High Priority' }]
+        };
+        
+        // when
+        const { emitted } = runNoolsLib(config);
+
+        // then
+        expect(emitted).to.have.length(2);
```

Comment:
It's a little weird that there are two emissions, but we are only asserting on the first emission. What is the second emission?